app/controllers/api/v1/search_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SearchController < Api::V1::BaseController
+      before_action do
+        require_access_token %w[read]
+      end
+
+      rescue_from ActionController::ParameterMissing, RuntimeError do
+        render json: I18n.t("api.endpoint_errors.search.cant_process"), status: :unprocessable_entity
+      end
+
+      def user_index
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for user_index is too high. [22.45/20]